skimage/segmentation/_expand_labels.py

```diff
@@ -0,0 +1,107 @@
+""" 
+expand_labels is  inspired by code in CellProfiler, 
+code licensed under BSD-3 licenses.
+Website: http://www.cellprofiler.org
+
+Copyright (c) 2020 Broad Institute
+All rights reserved.
+
+Original author/s: Cellprofiler team 
+"""
+
+import numpy as np
+from scipy.ndimage import distance_transform_edt
+
+
+def expand_labels(label_image, distance):
+    """Expand labels in label image by ``distance`` pixels without overlapping.
+
+    Given a label image, each label is grown by up to distance pixels.
+    However, where labels would start to overlap, the label growth may
+    stop at less than distance pixels (this is where it differs from a
+    morphological dilation, where a connected component with a high label
+    number can potentially override connected components with lower label
+    numbers).
+
+    Parameters
+    ----------
+    label_image : ndarray of dtype int
+        label image
+    distance : float
+        Number of pixels by which to grow the labels.
+
+    Returns
+    -------
+    enlarged_labels : ndarray of dtype int
+        Labeled array, where all connected regions have been enlarged
+
+    Notes
+    -----
+    This is equivalent to CellProfiler [1] [2] IdentifySecondaryObjects method
+    using the option "Distance-N".
+
+    The basic idea is that you have some seed labels that you want
+    to grow by n pixels to give a mask for a larger object.
```

Comment:
a repeat of the problematic description

Reply:
I reworded the whole docstring as there was some redundancy between the top part of the docstring and the `Notes` section. I think I made the description more exact and less hand-waving now, but I think this might be at the cost of being intuitively understandable. The code itself and the example result actual explain this better than I can with prose. Feel free to change the wording.